data/supporting_members.yaml

```diff
@@ -375,3 +375,140 @@ data:
       links:
         - icon: twitter
           url: https://twitter.com/NickyVdg
+    - name: Pieter Vander Vennet
+      nickname: pietervdvn
+      photo: https://zeus.ugent.be/zeuswpi/O_nr6ggA.jpg
+      quote: Is dat al gemapped?
+      description: Na de master Wiskundige informatica af te werken waarbij hij enkele jaren in het Zeus-bestuur heeft gezeten, komt hij nu nog graag eens langs om te kelderen of om het over open kaarten en routeplanning te hebben. Daarnaast is hij ook fan van een overdosis hackerspace en lichtjes, waarvoor hij ieder jaar naar Leipzig afzakt en Zeus-mensjes meeneemt daarnaartoe.
+      links:
+        - icon: map
+          url: https://osm.org/user/Pieter%20Vander%20Vennet
+        - icon: github
+          url: https://github.com/pietervdvn
+        - icon: twitter
+          url: https://twitter.com/pietervdvn
+    - name: Bart Coppens
+      nickname: bcoppens
+      photo: https://zeus.ugent.be/zeuswpi/lmKTWGfn.jpg
+      quote: |-
+        <WVDSCHEL> I AM BACK
+        <WVDSCHEL> ma nie tegen bcoppens zeggen, anders moet ik verderwerken
+      description: Houdt van security, assembly, en flitsende ASCII spiralen.
+      links:
+        - icon: twitter
+          url: https://twitter.com/bartcopp
+        - url: https://bartcoppens.be
+    - name: Kenneth Van den Driessche
+      nickname: tleilaxu
+      photo: https://zeus.ugent.be/zeuswpi/dictkelder.jpg
+      quote: TODO
+      description: Komt wel nog
+      links: []
+    - name: Michiel Van den Berghe
+      nickname: michilus
+      photo: https://zeus.ugent.be/zeuswpi/CznJ6WTT.jpg
+      quote: Ik zorg voor de camembert
+      description: Pythonista, space geek, zeppelinbouwer, Albertkanaalfanaat en oude zak met een e-paperfetish. Steekt geregeld elektronica in brand.
+      links:
+        - icon: github
+          url: https://github.com/mivdnber
+    - name: Felix Van der Jeugt
+      nickname: noctua
+      photo: https://zeus.ugent.be/zeuswpi/9_L03Pm8.jpg
+      quote: |-
+        <noctua> dwergen
+        <noctua> dwergen
+        <noctua> dwergen
+        <noctua> dwergen
+        <noctua> Sorry guys
+        <noctua> dwergen
+        <noctua> dwergen
+      description: Ik kom Ice Tea's halen in de kelder.
```

Comment:
Maybe replace with `Komt geen Ice Tea's meer halen in de kelder.` :-(

Reply:
:cry: